<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="Boton" name="WidgetFour" native="true">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="Boton" name="WidgetOne" native="true">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="Boton" name="WidgetFive" native="true">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Boton" name="WidgetTwo" native="true">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Boton" name="WidgetThree" native="true">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Script&quot;;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QWidget</extends>
   <header>boton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
